AndroidPayPalWebSdkIntegrationExample/app/src/main/res/layout/activity_on_payment_completion.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
+    tools:context=".OnPaymentCompletionActivity">
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fitsSystemWindows="true"
+        android:padding="@dimen/fab_margin">
+
+        <TextView
+            android:id="@+id/toolbar_title"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:fontFamily="sans-serif-medium"
+            android:gravity="center"
+            android:text="@string/paymentCompletionSuccess"
+            android:textColor="@color/material_dynamic_primary0"
+            android:textSize="25sp" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/paymentCompletionSuccessPayPalOrderId"
+            android:textSize="20sp" />
+
+        <TextView
+            android:id="@+id/payPalOrderId"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="... Waiting for Data ..."
+            android:textSize="20sp" />
+
+        <TextView
+            android:id="@+id/payPalOrderData"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@color/material_on_background_disabled"
+            android:gravity="left"
+            android:text="@string/payPalOrderDetailsLoading"
+            android:textSize="20sp" />
```

Comment:
We might want to move this content outside of the `AppBarLayout` as it usually has content that lives in the Toolbar. You could create a LinearLayout child view underneath the AppBarLayout where these views can be moved to.